queries.md

```diff
@@ -18,7 +18,7 @@ Limit: {20}
 ### 3. All the companies founded between 2000 and 2005, both years included. Retrieve only the `name` and `founded_year` fields.
 
 <!-- Your Code Goes Here -->
-Filter: { $and: [ {founded_year: { $gte: 2000 } }, {founded_year: { $lte: 2005 } }]}
+Filter: {founded_year: { $in:[2000,2005] } }}
```

Comment:
- **Query**: `{$and: [{founded_year: {$gte: 2000}}, {founded_year:{$lte: 2005}}]}`
- **Projection**: `{name: 1, _id: 0, founded_year: 1}`